ricecooker/utils/zip.py

```diff
@@ -47,24 +48,32 @@ def reader(x):
             return _read_file(os.path.join(path, x))
 
     # otherwise, if it's a zip file, open it up and pull out the list of names
-    elif os.path.isfile(path) and os.path.splitext(path)[1] == ".zip":
-        inputzip = zipfile.ZipFile(path)
-        paths = inputzip.namelist()
+    elif os.path.isfile(path):
+        extension = os.path.splitext(path)[1]
+        try:
+            inputzip = zipfile.ZipFile(path)
+            paths = inputzip.namelist()
 
-        def reader(x):
-            return inputzip.read(x)
+            def reader(x):
+                return inputzip.read(x)
 
-    else:
-        raise Exception("The `path` must either point to a directory or to a zip file.")
+        except Exception:
+            raise Exception(
+                "The `path` must either point to a directory or to a zip archive."
+            )
```

Comment:
I feel like I've been yelled at by a linter for trying to except a base `Exception` and had to make my own subclass of it to pass - is this due to lack of parity between Ricecooker and other repos on linting or am I misremembering?

Reply:
The only thing I can think of that a linter might shout at you for is a bare `except` clause without an `Exception` class or subclass specified.